vendor/wgpu/sdl2glue/glue_darwin.odin

```diff
@@ -0,0 +1,25 @@
+package wgpu_sdl2_glue
+
+import    "vendor:sdl2"
+import    "vendor:wgpu"
+import CA "vendor:darwin/QuartzCore"
+import NS "core:sys/darwin/Foundation"
+
+GetSurface :: proc(instance: wgpu.Instance, window: ^sdl2.Window) -> wgpu.Surface {
+    window_info: sdl2.SysWMinfo 
```

Comment:
Majority of this file doesn't use tabs for indentation, make sure to run it through a `-vet -strict-style -vet-tabs -disallow-do` to adhere to our library standards